.github/workflows/gradle.yml

```
@@ -1,26 +1,29 @@
-name: Java CI
+name: Java 17 (MC 1.17+)
 
 on: [push, pull_request]
 
 jobs:
     build:
         if: "! contains(toJSON(github.event.commits.*.message), '[ci skip]')"
         runs-on: ubuntu-latest
-
         steps:
             - uses: actions/checkout@v2
```

Let's bump all actions at once
```suggestion
            - uses: actions/checkout@v3
```